### PR TITLE
Laravel 5.6: What's New?

### PR DESCRIPTION
**Pull request by unicodeveloper**

### CODE REVIEW

File: _posts/2018-02-20-laravel56-release-what-is-new.markdown [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+      'username' => 'Laravel Log',
+      'emoji' => ':boom:',
+      'level' => 'critical',
+  ],
+],
+```
+
+The `stack` driver allows you to combine multiple channels, in this case, `syslog` and `slack`, into a single log channel as shown above.
+
+Furthermore, with the `tap` array on a channel's configuration, you can easily customize Monolog for an existing channel like so:
```
Comment (lukeoliff) [inline _posts/2018-02-20-laravel56-release-what-is-new.markdown]: Perhaps elaborate on what Monolog is

File: _posts/2018-02-20-laravel56-release-what-is-new.markdown [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+
+I covered all the [major additions and deprecations of Bootstrap 4 in this article](https://auth0.com/blog/whats-new-in-bootstrap4).
+
+## Deprecations and Other Updates
+
+* Generating an API resource controller can now be done by using the `--api` switch when executing the `make:controller` command.
+* Laravel 5.6 introduced helper methods, `Arr::wrap()`, `classes_uses_recursive()`, `Str::uuid()`, and `Str::orderedUuid()`, the latter generating a timestamp first UUID that's more easily and efficiently indexed by databases like MySQL.
+* PHPUnit, upgraded to v7.
+* Laravel Mix, upgraded to v2.
+* The deprecated Artisan`optimize` command has been removed.
```
Comment (lukeoliff) [inline _posts/2018-02-20-laravel56-release-what-is-new.markdown]: Artisan`optimize`  -> Artisan `optimize`

File: _posts/2018-02-20-laravel56-release-what-is-new.markdown [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+
+---
+
+Laravel is a free, open-source PHP framework designed for building web applications with an expressive and elegant syntax. Laravel has a high level of abstraction which shields the common developer from complex inner workings. Laravel saves you time and effort because it ships with a lot of features out of the box. Without further ado, let's dive right into **Laravel 5.6**.
+
+## What's new in Laravel 5.6?
+
+### 1. Support For Argon Password Hashing
+
+Argon2, the recommended password hashing algorithm by the Password Hashing Competition, is a modern algorithm for securely hashing passwords. And it comes in two distinct flavours, Argon 2i and Argon 2d. PHP 7.2 recently added support for Argon 2i password hashing. Therefore, [Michael Lundbol](https://github.com/morloderex) took the initiative to add support for Argon hashing in Laravel.
```
Comment (lukeoliff) [inline _posts/2018-02-20-laravel56-release-what-is-new.markdown]: flavours -> flavors

File: _posts/2018-02-20-laravel56-release-what-is-new.markdown [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+- 2017-07-29-laracon-summary-day-2
+---
+
+---
+
+**TL;DR:** On Wednesday, February 7, 2018, **Laravel 5.6**, was released to the public. It's a major release to the most popular PHP framework on GitHub as of this writing. Furthermore, [Spark 6.0](https://spark.laravel.com) was also released alongside **Laravel 5.6**. According to the policy laid down by the Taylor and the core team, major Laravel framework releases are released every six months (February and August). In this article, I'll cover the new features in Laravel 5.6 and several other changes and deprecations.
+
+---
+
+Laravel is a free, open-source PHP framework designed for building web applications with an expressive and elegant syntax. Laravel has a high level of abstraction which shields the common developer from complex inner workings. Laravel saves you time and effort because it ships with a lot of features out of the box. Without further ado, let's dive right into **Laravel 5.6**.
```
[the remark this answers is not in the document]
Comment (unicodeveloper) [reply _posts/2018-02-20-laravel56-release-what-is-new.markdown]: Lol is that a thing? I wrote the content so I took out a paragraph that defines Laravel in the most simple and expressive form.

File: _posts/2018-02-20-laravel56-release-what-is-new.markdown (hunk 1 above)
Comment (unicodeveloper) [reply _posts/2018-02-20-laravel56-release-what-is-new.markdown]: I'll add a small note about Monolog. Thanks

**Pull request merged**